game/src/plugins/skills/fishing/src/fishing_data.kt

```diff
@@ -0,0 +1,42 @@
+package org.apollo.game.plugin.skills.fishing
+
+import org.apollo.game.model.Animation
+import java.util.*
+
+
+data class Fish(val id: Int, val level: Int, val exp: Double, val name: String)
+data class FishingTool(val id: Int, val animation: Animation, val message: String, val bait: Int, val name: String)
+data class FishingSpot(val id: Int, val tools:IntArray, val firstFish: IntArray, val secondFish: IntArray)
+
+val FISH = mutableMapOf<Int, Fish>(
+        317 to Fish(317, 1, 10.0, "shrimp"),
```

Comment:
I think we can lookup the item name from the ID here instead of declaring it manually.